### PR TITLE
4908 - additional scenarios

### PR DESCRIPTION
**Pull request by milosdes**

### CODE REVIEW

**Review by nkan-aot2 — APPROVED** [review]

**Review by aparna-aot — APPROVED** [review]

File: web/src/components/FOI/Home/Redlining.js [hunk 1]
```diff
@@ -121,11 +121,13 @@ const Redlining = React.forwardRef(
     const [docViewerMath, setDocViewerMath] = useState(null);
     const [docInstance, setDocInstance] = useState(null);
     const [modalOpen, setModalOpen] = useState(false);
+    const [messageModalOpen, setMessageModalOpen] = useState(false)
```
Comment (milosdes) [inline web/src/components/FOI/Home/Redlining.js:124]: Edit this

File: web/src/components/FOI/Home/Redlining.js [hunk 2]
```diff
@@ -2037,6 +2042,8 @@ const Redlining = React.forwardRef(
 
     const cancelRedaction = () => {
       setModalOpen(false);
+      setMessageModalOpen(false);
+      setSelectedPageFlagId(null);
```
Comment (milosdes) [inline web/src/components/FOI/Home/Redlining.js:2046]: Added this

File: web/src/components/FOI/Home/Redlining.js [hunk 3]
```diff
@@ -2068,12 +2075,34 @@ const Redlining = React.forwardRef(
...
+          setModalMessage(
+          <div>You have 'Not Responsive' selected as a default section.
+            <ul>
+              <li className="modal-message-list-item">To flag this page as 'Withheld in Full', remove the default section.</li>
+              <li className="modal-message-list-item">To flag this full page as 'Not Responsive', use the 'Not Responsive' page flag.</li>
+            </ul>
+          </div>);
+          setMessageModalOpen(true)
+    }
+
```
Comment (milosdes) [inline web/src/components/FOI/Home/Redlining.js:2089]: Added

File: web/src/components/FOI/Home/Redlining.js [hunk 4]
```diff
@@ -3754,6 +3812,41 @@ const Redlining = React.forwardRef(
...
+          </DialogContent>
+          <DialogActions className="foippa-modal-actions">
+            <button
+              className="btn-bottom btn-cancel"
+              onClick={cancelRedaction}
+            >
+              Cancel
+            </button>
+          </DialogActions>
+        </ReactModal>
```
Comment (milosdes) [inline web/src/components/FOI/Home/Redlining.js:3849]: Added modal

**Pull request merged**